### DISCUSSION TITLE
Idea for use in some components of the Elemente Plus

### DISCUSSION THREAD

**Discussion by leeobrum** [element-plus/element-plus#16191] [Ideas]
What do you think of this idea, to add this behavior to some components? It's something very up-to-date these days.

The example works on mobile devices. To test you need to add browser emulation.

[Link Exemple](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6IjxzY3JpcHQgc2V0dXAgbGFuZz1cInRzXCI+XG5pbXBvcnQgeyByZWYsIG9uTW91bnRlZCwgdmVyc2lvbiBhcyB2dWVWZXJzaW9uIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgdmVyc2lvbiBhcyBlcFZlcnNpb24gfSBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBFbGVtZW50UGx1cyB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuXG5jb25zdCBjYXJkID0gcmVmPEhUTUxFbGVtZW50IHwgbnVsbD4obnVsbClcbmNvbnN0IGlzRHJhZ2dpbmcgPSByZWYoZmFsc2UpXG5jb25zdCBzdGFydFBvc2l0aW9uWCA9IHJlZigwKVxuY29uc3QgY3VycmVudFRyYW5zbGF0ZVggPSByZWYoMClcblxub25Nb3VudGVkKCgpID0+IHtcbiAgY2FyZC52YWx1ZSA9IGRvY3VtZW50LnF1ZXJ5U2VsZWN0b3IoJy5wb2ludGVyLWNhcmQnKVxufSlcblxuY29uc3QgaGFuZGxlTW91c2VEb3duID0gKF9ldmVudDogTW91c2VFdmVudCkgPT4ge1xuICAvKiBpZiAoY2FyZC52YWx1ZSkge1xuICAgIHN0YXJ0UG9zaXRpb25YLnZhbHVlID0gZXZlbnQuY2xpZW50WFxuICAgIGlzRHJhZ2dpbmcudmFsdWUgPSB0cnVlXG4gIH0gKi9cbn1cbmNvbnN0IGhhbmRsZU1vdXNlTW92ZSA9IChfZXZlbnQ6IE1vdXNlRXZlbnQpID0+IHtcbiAgLyogaWYgKGlzRHJhZ2dpbmcudmFsdWUgJiYgY2FyZC52YWx1ZSkge1xuICAgIGNvbnNvbGUubG9nKGNhcmQudmFsdWUpXG4gICAgY29uc29sZS5sb2coaXNEcmFnZ2luZy52YWx1ZSlcbiAgICBjb25zdCBkaXN0YW5jZVggPSBldmVudC5jbGllbnRYIC0gc3RhcnRQb3NpdGlvblgudmFsdWVcbiAgICBjdXJyZW50VHJhbnNsYXRlWC52YWx1ZSA9IE1hdGgubWluKE1hdGgubWF4KDAsIGRpc3RhbmNlWCksIDEwMClcbiAgICBjYXJkLnZhbHVlLnN0eWxlLnRyYW5zZm9ybSA9IGB0cmFuc2xhdGVYKCR7Y3VycmVudFRyYW5zbGF0ZVgudmFsdWV9JSlgXG4gIH0gKi9cbn1cbmNvbnN0IGhhbmRsZU1vdXNlVXAgPSAoKSA9PiB7XG4gIC8qIGlmIChpc0RyYWdnaW5nLnZhbHVlICYmIGNhcmQudmFsdWUpIHtcbiAgICBpc0RyYWdnaW5nLnZhbHVlID0gZmFsc2VcbiAgICBjdXJyZW50VHJhbnNsYXRlWC52YWx1ZSA9IDBcbiAgICBjYXJkLnZhbHVlLnN0eWxlLnRyYW5zZm9ybSA9ICcnXG4gIH0gKi9cbn1cbmNvbnN0IGhhbmRsZVRvdWNoU3RhcnQgPSAoZXZlbnQ6IFRvdWNoRXZlbnQpID0+IHtcbiAgaWYgKGNhcmQudmFsdWUpIHtcbiAgICBzdGFydFBvc2l0aW9uWC52YWx1ZSA9IGV2ZW50LnRvdWNoZXNbMF0uY2xpZW50WFxuICAgIGlzRHJhZ2dpbmcudmFsdWUgPSB0cnVlXG4gIH1cbn1cbmNvbnN0IGhhbmRsZVRvdWNoTW92ZSA9IChldmVudDogVG91Y2hFdmVudCkgPT4ge1xuICBpZiAoaXNEcmFnZ2luZy52YWx1ZSAmJiBjYXJkLnZhbHVlKSB7XG4gICAgY29uc3QgZGlzdGFuY2VYID0gZXZlbnQudG91Y2hlc1swXS5jbGllbnRYIC0gc3RhcnRQb3NpdGlvblgudmFsdWVcbiAgICBjb25zdCBtYXhUcmFuc2xhdGVYID0gMzU7IC8vIERlZmluaXIgYSBwb3JjZW50YWdlbSBtw6F4aW1hIGRlIG1vdmltZW50b1xuICAgIGN1cnJlbnRUcmFuc2xhdGVYLnZhbHVlID0gTWF0aC5taW4oTWF0aC5tYXgoMCwgZGlzdGFuY2VYKSwgbWF4VHJhbnNsYXRlWClcbiAgICBjYXJkLnZhbHVlLnN0eWxlLnRyYW5zZm9ybSA9IGB0cmFuc2xhdGVYKCR7Y3VycmVudFRyYW5zbGF0ZVgudmFsdWV9JSlgXG4gICAgaWYgKGN1cnJlbnRUcmFuc2xhdGVYLnZhbHVlID09PSBtYXhUcmFuc2xhdGVYKSB7XG4gICAgICBpc0RyYWdnaW5nLnZhbHVlID0gZmFsc2UgLy8gUGFyYXIgZGUgYXJyYXN0YXIgcXVhbmRvIGF0aW5naXIgYSBwb3JjZW50YWdlbSBtw6F4aW1hXG4gICAgfVxuICB9XG59XG5jb25zdCBoYW5kbGVUb3VjaEVuZCA9ICgpID0+IHtcbiAgaWYgKGlzRHJhZ2dpbmcudmFsdWUgJiYgY2FyZC52YWx1ZSkge1xuICAgIGlzRHJhZ2dpbmcudmFsdWUgPSBmYWxzZVxuICAgIGN1cnJlbnRUcmFuc2xhdGVYLnZhbHVlID0gMFxuICAgIGNhcmQudmFsdWUuc3R5bGUudHJhbnNmb3JtID0gJydcbiAgfVxufVxuPC9zY3JpcHQ+XG5cbjx0ZW1wbGF0ZT5cbiAgPGVsLWNhcmRcbiAgICBjbGFzcz1cInBvaW50ZXItY2FyZFwiXG4gICAgcmVmPVwiY2FyZFwiXG4gICAgQHRvdWNoc3RhcnQ9XCJoYW5kbGVUb3VjaFN0YXJ0XCJcbiAgICBAdG91Y2htb3ZlPVwiaGFuZGxlVG91Y2hNb3ZlXCJcbiAgICBAbW91c2Vkb3duPVwiaGFuZGxlTW91c2VEb3duXCJcbiAgICBAbW91c2Vtb3ZlPVwiaGFuZGxlTW91c2VNb3ZlXCJcbiAgICBAdG91Y2hlbmQ9XCJoYW5kbGVUb3VjaEVuZFwiXG4gICAgQG1vdXNldXA9XCJoYW5kbGVNb3VzZVVwXCJcbiAgPlxuICAgIDxkaXYgY2xhc3M9XCJjYXJkLWNvbnRlbnRcIj5cbiAgICAgIDwhLS0gQ29udGXDumRvIGRvIGNhcmQgLS0+XG4gICAgICA8cD5Db250ZcO6ZG8gZG8gY2FyZCBhcXVpLi4uPC9wPlxuICAgIDwvZGl2PlxuICA8L2VsLWNhcmQ+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJ0c2NvbmZpZy5qc29uIjoie1xuICBcImNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJtb2R1bGVcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVJlc29sdXRpb25cIjogXCJCdW5kbGVyXCIsXG4gICAgXCJ0eXBlc1wiOiBbXCJlbGVtZW50LXBsdXMvZ2xvYmFsLmQudHNcIl0sXG4gICAgXCJhbGxvd0ltcG9ydGluZ1RzRXh0ZW5zaW9uc1wiOiB0cnVlLFxuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlXG4gIH0sXG4gIFwidnVlQ29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiAzLjNcbiAgfVxufVxuIiwiX28iOnt9fQ==)

**Comment by xxholly32** [comment]
Sometimes, excessive ellipses can actually hinder readability.
